docs/rules/OrganizeImports.md

```diff
@@ -1328,3 +1328,86 @@ object RemoveUnused {
   val long: JLong = JLong.parseLong("0")
 }
 ```
+
+`targetDialect`
+--------------
+
+Specifies the [wildcards and renames](https://docs.scala-lang.org/scala3/reference/changed-features/imports.html)
+syntax used when imports are coalesced or rewritten.
+
+### Value type
+
+Enum: `Auto | Scala2 | Scala3`
+
+#### `Auto`
+
+Infer the dialect from compilation settings (Scala version or `-Xsource` when
+provided). This is safe only for sources that are not cross-compiled (see
+rationale below).
+
+#### `Scala2`
+
+Use `_` as wildcard and `=>` for renames.
+
+#### `Scala3`
+
+Use `*` as wildcard and `as` for renames.
+
+### Default value
+
+`Scala2`
+
+Rationale: `Auto` is not a safe default for projects cross-compiling with
+Scala 3.x and Scala 2.x without `-Xsource:3`, as the Scala 3.x Scalafix run
+would introduce Scala 2.x compilation errors.
```

Comment:
It's really a shame we can't default to `Auto` :disappointed: 

@tgodzik maybe metals has more information and could force `Auto` whenever we know for sure that sources won't be cross-compiled with Scala2? I am not convinced as we would get a lot of false negatives with the checks I have in mind
- source files rooted in a `scala-3` directory
- source files using other Scala 3 features (I am not sure we can pull this information for the tree - as far as I know the only way would be to try and fail to parse with a Scala 2 dialect...)

We could do something similar in sbt-scalafix (although this would be the first time we mingle with rule configuration) with fairly low risk when detecting that we both have
- no usage of sbt-projectmatrix
- no `crossScalaVersions` set

Reply:
I don't think we have that information even in Metals especially that with crossScalaVersion only one version would be imported. 

The alternative to Auto would be to have StandardLayout which would do Scala2 target under `scala` and `scala-2`, Scala3 for `scala-3` directories.

Anyway, I think having Scala2 as default makes sense

Reply:
It turned out `StandardLayout` wasn't so hard to implement so I added it and made it the default. Thanks :+1:

Reply:
Awesome! Thanks!